Ignore properties that are already suffixed (#208)
fullstorydev/fullstory-data-layer-observer: docs/operator_convert.md

| Option           | Type                   | Default     | Description                                                                                                                                                                                                         |
|------------------|------------------------|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `enumerate`*     | `boolean`              | `false`     | Automatically converts all string values into their numeric equivalent.                                                                                                                                             |
| `ignore`         | `string` or `string[]` | `undefined` | Can be paired with `enumerate` option to ignore one or more properties when performing the `enumerate`.                                                                                                             |

> | `ignoreSuffixed` | `boolean` | `true`      | Can be paired with `enumerate` option to ignore any properties that are suffixed when performing the `enumerate`.  [Suffixes](https://help.fullstory.com/hc/en-us/articles/360020623234#property-name-requirements) |

Reviewer: ```suggestion
| `ignoreSuffixed` | `boolean` | `true`      | Can be paired with `enumerate` option to ignore any properties with [known type suffixes](https://help.fullstory.com/hc/en-us/articles/360020623234#property-name-requirements) when performing the `enumerate`. |
```

 }
]
```
## Automatically convert all "numeric" strings ignoring suffixed property

### Rule

```javascript
{
 source: 'digitalData.cart.price[(userId, available,basePrice,cartTotal,priceWithTax)]',
 operators: [

>    { name: 'convert', enumerate: true, ignoreSuffixed: true }, 

Reviewer: I guess we could omit `ignoreSuffixed` now that ignoring suffixes is the default.

Author: > LGTM. Left a couple small things to consider, but nothing required to merge this change.

Made those changes.
